### ISSUE TITLE
 Quaternary geology of the Nappanee West 7.5-minute quadrangle, Indiana

### ISSUE THREAD

**Issue by ericjohannes** [UW-Macrostrat/burwell-processing#426]
**Name:**

Fleming, A. H., and Karaffa, M., 2012, Quaternary geology of the Nappanee West 7.5-minute quadrangle, Indiana: Indiana Geological Survey Miscellaneous Map 89, scale 1:24,000

**Source URL:**

https://iu.box.com/s/8o5hxhq4uh98l923ab1nzmag751vl6ia

**Estimated scale:**

large

**Folder name:**

**Dataset names:**

NappaneeWest_SurficialGeology.gdb/NappaneeWest_SurficialGeology

**Number of bedrock polygons:**
52 (surficial polygons)

**Lithology field:**

**Time field:**

Age

**Stratigraphy name field:**

**Description field:**

Name
UnitDescription

**Comments field:**

IceLobe
FormSequen

**Comments:**

## Todo:

- [ ] Insert new source record
- [ ] Add data to database
- [ ] Process and add to homogenized tables
- [ ] Process and import lines
- [ ] Process and import points
- [ ] Match
- [ ] Roll tiles

**Comment by aazaff** [comment]
@cvavrus this one looks good, but I'm not sure what should they change Late Glacial to? It doesn't seem to be a recognized Macrostrat Int.

**Comment by cvavrus** [comment]
@aazaff my guess would be that it could be changed to Holocene, but I'm not 100% sure on that

**Comment by ericjohannes** [comment]
I'll ask one of our geologists about it

**Comment by ericjohannes** [comment]
@aazaff @cvavrus the answer I got back was "Late Pleistocene." I can update the map and give a new download link for it

**Comment by ericjohannes** [comment]
Here's the updated gdb https://iu.box.com/s/8o5hxhq4uh98l923ab1nzmag751vl6ia

**Comment by aazaff** [comment]
Yes, Late Pleistocene makes more sense than Holocene. I edited the link to the updated gdb to the primary issue.

I think this one is ready for @cvavrus to pre-process and add to the queue.

**Comment by cvavrus** [comment]
@ericjohannes the unitdescri column seems to be cut short for this map as well

**Comment by ericjohannes** [comment]
@cvavrus when I download this gdb https://iu.box.com/s/8o5hxhq4uh98l923ab1nzmag751vl6ia and open it up, the UnitDescription column looks OK to me. The strings are 400-1000 characters long and look to all end in periods like all the text is there.

**Comment by cvavrus** [comment]
@ericjohannes that may have been my mistake, I haven't worked with many gdb files yet. When I download the folder I get a number of files that begin with a0000000 and some number, do you know which is the correct one?

**Comment by cvavrus** [comment]
@ericjohannes nevermind, I think I figured it out

**Comment by ericjohannes** [comment]
@cvavrus sounds good

**Comment by cvavrus** [comment]
**Folder name:**
in_napp

Suggested insert:
```
INSERT INTO maps.large (orig_id,source_id,name,age,descrip,strat_name,t_interval,b_interval,comments,geom) 
SELECT gid,1,formsequen,age,descrip,strat_name,late_id,early_id,icelobe,ST_MakeValid(geom) FROM sources.in_napp where geom is not null;
```
